src/thirdparty/axom/sol.hpp

```diff
@@ -11468,11 +11468,13 @@ namespace stack {
 	struct popper {
 		inline static decltype(auto) pop(lua_State* L) {
 			record tracking{};
-#ifdef __INTEL_COMPILER
-			auto&& r = get<T>(L, -lua_size<T>::value, tracking);
-#else
+// AXOM EDIT START: sol.hpp(11477): error: an rvalue reference cannot be bound to an lvalue

```

Comment:
This is to fix intel@19 with C++14. I could not get intel@18 to work at all with C++14.  If I remember correctly this toggled back and forth between 18 and 19 and since noone uses sol with intel@18, I decided to give up.  This was not the only error with intel@18.

Reply:
Thanks. 

It's a bit difficult to see from github, but it looks like we no longer have an axom-specific patch here for sol. If so, should we remove these comment lines and the associated lines from the patch file? (or am I missing something?)

Reply:
This is an axom specific patch, and it has the markings. There is also a patch file.